src/modules/relational-db/relational-db.module.ts

```diff
@@ -0,0 +1,12 @@
+import { ContainerModule } from 'inversify'
+import type { RelationalDbClient } from './create-relational-db-client'
+import { createRelationalDbClient, RELATIONAL_DB_CLIENT } from './create-relational-db-client'
+
+export const DATABASE = process.env.DB_NAME || 'roadoxe'
+
+export const relationalDbModule = new ContainerModule((bind, _unbind) => {
```

Comment:
eslint is not suppose to generate waring if a unused variable is prefixed with it maybe one of yo guys fix this in the eslint configurations @Shazil2154 @iamusmanazeem @Ismail-butt